editor/script_create_dialog.cpp

```diff
@@ -168,9 +161,9 @@ void ScriptCreateDialog::_load_exist() {
 	String path = file_path->get_text();
 	RES p_script = ResourceLoader::load(path, "Script");
 	if (p_script.is_null()) {
-		alert->get_ok()->set_text(TTR("Ugh"));
+		alert->get_ok()->set_text(TTR("OK"));
 		alert->set_text(vformat(TTR("Error loading script from %s"), path));
-		alert->popup_centered_minsize();
+		alert->popup_centered();
```

Comment:
Fixed. Used `EDSCALE` to scale controls properly to Editor size. Didn't changed `Alert` calls tho. `Alert` sets itself as `minsize` is it ok?

Also fixed missing `TTR`